server: Prettier Formatting

znewton opened the pull request:
## Description

Run pnpm run format in Gitrest, Historian, and Routerlicious codebases

Prompted by auto-formatting headache in #14401

znewton committed:
Run pnpm run format in server codebases

znewton committed:
add prettier configs to make docker build lint happy

tylerbutler commented:
@znewton Thanks for doing this! Based on the lint failure, you may need to adjust the eslint config to extend prettier, or update to the 2.0.0 version of the fluid shared lint config.

znewton committed:
fix lambdas-driver linting

znewton commented:
@tylerbutler thanks for the hint! There was 1 (of 50+) package in R11s with a missing "prettier" extension 😆

jikim-msft reviewed (approved):
Besides the comment that @tylerbutler left above, everything else looks good! Thank you again for working on this!

tylerbutler commented:
You need to update the root package.json for each release group and add the prettier/prettier:fix scripts to the lint scripts. otherwise there's no CI protection.

znewton committed:
formatting moved some lint disable comments

znewton committed:
Merge branch 'main' of github.com:microsoft/FluidFramework into server/formatting

znewton committed:
Server specific prettierignore for docker

znewton committed:
formatting

znewton commented on server/historian/packages/historian-base/src/utils.ts:
```diff
@@ -16,80 +16,80 @@
+		}
+		const encoded = Buffer.from(base64TokenMatch[1], "base64").toString();
+
+		const tokenMatch = encoded.match(/(.+):(.+)/);
```
This was not introduced in this PR

znewton commented:
@tylerbutler I added the prettier, prettier:fix, prettier:root, and prettier:root:fix scripts to gitrest/package.json, historian/package.json, and r11s/package.json. I also added gitrest/.prettierignore, historian/.prettierignore, and r11s/.prettierignore so that internal package ignore paths were pointing at an actual file. I couldn't reuse the FF/.prettierignore because I wanted to make sure each Docker build pulled in the appropriate prettierignore

znewton committed:
Merge branch 'main' of github.com:microsoft/FluidFramework into server/formatting

znewton committed:
more moved disable comments
